lib/livebook/zta/tailscale.ex

```diff
@@ -0,0 +1,107 @@
+defmodule Livebook.ZTA.Tailscale do
+  @doc """
```

Comment:
```suggestion
  @moduledoc """
```

Reply:
I think I copied another ZTA module which also has this so they should probably all be changed :)